File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    
    for q in quiz:
        x, op, y, _, z = q.split(' ');
        res = int(x) + int(y) if op == '+' else int(x) - int(y)
        answer.append("O" if res == int(z) else "X")
    
    return answer

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    let answer = [];
    
    for (const q of quiz) {
        const [x, op, y, _, z] = q.split(' ');
        const res = op === '+' ? Number(x) + Number(y) : Number(x) - Number(y);
        answer.push(res === Number(z) ? "O" : "X");
    }
    
    return answer;
}